app/controllers/events/view/tickets/access-codes/index.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  columns: [
+    {
+      propertyName : 'code',
+      title        : 'Access Code'
+    },
+    {
+      propertyName : 'accessUrl',
```

Comment:
Models-table uses these in query, we must use the correct propertyName here, Please make it `access-url`.